src/sql/magic.py

```diff
@@ -22,6 +22,7 @@
 import sql.connection
 import sql.parse
 import sql.run
+from sql.parse import _option_strings_from_parser
```

Comment:
you're mixing stuff from different PRs. please open a PR that only modifies the requested file

Reply:
Fix it now